flow/connectors/mysql/validate.go

```diff
@@ -23,23 +23,30 @@ func (c *MySqlConnector) CheckSourceTables(ctx context.Context, tableNames []*ut
 	return nil
 }
 
+func checkGrantCommand(command []byte) error {
+	grant := shared.UnsafeFastReadOnlyBytesToString(command)
+	if strings.Contains(grant, "REPLICATION SLAVE") || strings.Contains(grant, "REPLICATION CLIENT") {
```

Comment:
I did not want this to be done, since string comparison could lead to incomplete permission comparison
need to find an alternative query

Reply:
fuzzy check in validation is fine as long as it's complete rather than correct